common/fileview2/fvwugen.cpp

```diff
@@ -112,7 +112,7 @@ IHqlExpression * buildDiskFileViewerEcl(const char * logicalName, IHqlExpression
     OwnedHqlExpr newRecord = createRecord(fields);
     newRecord.setown(createSymbol(createIdAtom("_SourceRecord_"), newRecord.getLink(), ob_private));
 
-    OwnedHqlExpr dataset = createNewDataset(createConstant(logicalName), newRecord.getLink(), createValue(no_thor), NULL, NULL, NULL);
+    OwnedHqlExpr dataset = createNewDataset(createConstant(logicalName), newRecord.getLink(), createValue(no_thor, makeNullType()), NULL, NULL, NULL);
```

Comment:
Should NULL be replaced by nullptr?

Reply:
Long term yes, but it should be a separate PR to replace NULL with nullptr.  Possibly throughout the source.